### PR TITLE
[docs] add two survey papers to readme

### LINKED ISSUE #159: [Update] - Move two new survey papers into  "最近更新", which should be located before '论文复现小组'
## **Detailed Description**

**Content Name/Link**: [State of the Art on Diffusion Models for Visual Computing](http://arxiv.org/abs/2310.07204)

**Current Status/Issue**: The document may not include the latest advancements in diffusion models for visual computing.

**Update Details**: The "State of the Art on Diffusion Models for Visual Computing" paper will be incorporated.

## Update

**I think we could add these two new papers into "最近更新" to extract more attention for our SoraSurvey Team.**

**What's more,  "最近更新"  should also be moved to a place near the top part of README.md. For example, before '论文复现小组'**

## **Additional Information**
**Reason for Update**:  This paper provides an intuitive starting point to explore video Diffusion model topic for researchers, artists, and practitioners alike.

### PR DESCRIPTION
**Pull request by seifer08ms**
### Summary
This pull request updates the README file with the latest research on diffusion models for visual computing, including the state-of-the-art paper and a comprehensive survey.

### Changes
- Incorporated  [arxiv 23 Paper](http://arxiv.org/abs/2310.07204) "State of the Art on Diffusion Models for Visual Computing" and  [CSUR 24 Paper](https://dl.acm.org/doi/10.1145/3626235)" Diffusion Models: A Comprehensive Survey of Methods and Applications", along with the respective GitHub repo [Github](https://github.com/YangLing0818/Diffusion-Models-Papers-Survey-Taxonomy).
- Ensured that the links are functional and the information is up-to-date.

### Review Notes
- Please review the added content for accuracy and formatting.
- Check that the links provided are correct and lead to the intended resources.
- **Please check if both the Chinese and English READMEs have been updated simultaneously.**

### Related Issues
- Closes #159 

### Additional Comments
- The inclusion of these resources will greatly benefit the community by providing a wealth of information on the latest advancements and applications in diffusion models for visual computing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,8 @@ Mini Sora 开源社区定位为由社区同学自发组织的开源社区（**
 | 7) Video Generation Task in Papers With Code |  [Link](https://paperswithcode.com/task/video-generation) |
 | 8) Sora: A Review on Background, Technology, Limitations, and Opportunities of Large Vision Models |  [**Paper**](https://arxiv.org/abs/2402.17177), [Github](https://github.com/lichao-sun/SoraReview), [中文翻译](https://mp.weixin.qq.com/s/KsmRg7SyCpRs7Bf3D48D2A)|
 | 9) Open-Sora-Plan (PKU-YuanGroup) |  [Github](https://github.com/PKU-YuanGroup/Open-Sora-Plan) |
+| 10) State of the Art on Diffusion Models for Visual Computing | [Paper](http://arxiv.org/abs/2310.07204)  |
+| 11) Diffusion Models: A Comprehensive Survey of Methods and Applications | [**CSUR 24 Paper**](https://dl.acm.org/doi/10.1145/3626235), [Github](https://github.com/YangLing0818/Diffusion-Models-Papers-Survey-Taxonomy) |
```
Comment (JunyaoHu) [inline README.md:175]: https://arxiv.org/abs/2209.00796

File: README_EN.md [hunk 2]
```diff
@@ -120,6 +120,8 @@ The Mini Sora open-source community is positioned as a community-driven initiati
 | 7) Video Generation Task in Papers With Code |  [Task](https://paperswithcode.com/task/video-generation) |
 | 8) Sora: A Review on Background, Technology, Limitations, and Opportunities of Large Vision Models |  [**Paper**](https://arxiv.org/abs/2402.17177), [Github](https://github.com/lichao-sun/SoraReview) |
 | 9) Open-Sora-Plan (PKU-YuanGroup) |  [Github](https://github.com/PKU-YuanGroup/Open-Sora-Plan) |
+| 10) State of the Art on Diffusion Models for Visual Computing | [Paper](http://arxiv.org/abs/2310.07204)  |
+| 11) Diffusion Models: A Comprehensive Survey of Methods and Applications | [**CSUR 24 Paper**](https://dl.acm.org/doi/10.1145/3626235), [Github](https://github.com/YangLing0818/Diffusion-Models-Papers-Survey-Taxonomy) |
```
Comment (JunyaoHu) [inline README_EN.md:124]: https://arxiv.org/abs/2209.00796

File: README.md (hunk 1 above)
Comment (JunyaoHu) [reply README.md:175]: arXiv version is better, please see https://github.com/mini-sora/minisora/issues/101#issuecomment-1972167603

File: README_EN.md (hunk 2 above)
Comment (JunyaoHu) [reply README_EN.md:124]: arXiv version is better, please see https://github.com/mini-sora/minisora/issues/101#issuecomment-1972167603

**Review by JunyaoHu — CHANGES_REQUESTED** [review]

**Comment by seifer08ms** [comment]
FYI : The acronym for ACM Computing Surveys is [CSUR](https://dl.acm.org/journal/csur).

File: README_EN.md (hunk 2 above)
Comment (seifer08ms) [reply README_EN.md:124]: > arXiv version is better, please see [#101 (comment)](https://github.com/mini-sora/minisora/issues/101#issuecomment-1972167603)

Okay, I'll update it

**Review by chg0901 — APPROVED** [review]

**Review by chg0901 — APPROVED** [review]

**Review by JunyaoHu — APPROVED** [review]

**Pull request merged**